library/gui/screen/src/main/java/org/quiltmc/qsl/screen/mixin/client/KeyboardMixin.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2016, 2017, 2018, 2019 FabricMC
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.screen.mixin.client;
+
+import org.quiltmc.qsl.screen.api.client.ScreenKeyboardEvents;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.client.Keyboard;
+import net.minecraft.client.gui.screen.Screen;
+
+@Mixin(Keyboard.class)
+abstract class KeyboardMixin {
+	// private synthetic method_1454(ILnet/minecraft/client/gui/screen/Screen;[ZIII)V
```

Comment:
i assume this is a lambda? add a comment mentioning which method calls this lambda, and roughly where

Reply:
> 

Yes... and i will probably not be able to do this. But i will try.

Reply:
You have to view the bytecode, which doesn't require genSources at all. I do expect comments to be here before merge, so please ask if you need help.

Reply:
Fixed.